test/simulations/PowerPerturbation.jl

```diff
@@ -10,8 +10,38 @@ using PowerDynamics: PowerPerturbation, simulate, SwingEqLVS, StaticLine, PowerG
     sol = simulate(PowerPerturbation(
         fraction = 0.9,
         node_number = 1,
-        tspan_fault = (0.1,1)),
+        tspan_fault = (0.1,1),
+        power_symbol = :P),
         grid, state, (0., 1.))
     @test sol != nothing
     @test sol.dqsol.retcode == :Success
 end
+
+@testset "test PowerPerturbation simulation invalid P should throw PowerPerturbationError" begin
+    invalidP = 1
```

Comment:
@janlisse: do you think we can't simply convert from integer to float? Isn't it kind of unhandy for users to be restricted to floats? Also, we would have to clearly document at every node type to use floats only.